agent/utils/utils.go

```diff
@@ -205,3 +205,9 @@ func SearchStrInDir(dir, filePrefix, content string) error {
 
 	return nil
 }
+
+// GetTaskID retrieves the task ID from task ARN.
+func GetTaskID(taskARN string) string {
+	fields := strings.Split(taskARN, "/")
+	return fields[len(fields)-1]
+}
```

Comment:
Can we add a validation here to ensure that taskARN is in correct formatting, the most basic one I can think of is len(fields) is of at least length 2.

Reply:
yeah, updated with some checks